cowamm/profitability/competitor_kpis/pancakeswap/pancakeswap_largest_pools_4232597.sql

```diff
@@ -0,0 +1,64 @@
+-- Computes the TVL for every Pancakeswap pool
+-- Then returns the top {{number_of_pools}} pools by TVL
```

Comment:
Why does curve not have `number_of_pools` (can we make it consistent) and also it seems this value is never set.